plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -292,6 +292,9 @@ public ExecutionResult executeInVR(final String routerIp, final String script, f
         if (details == null) {
             details = parser.getLines();
         }
+
+        s_logger.debug("Executing script in VR " + script);
```

Comment:
Please wrap in a ``if (s_logger.isDebugEnabled())`` block to avoid unnecessary string creation due to pre-method string concatenation when ``DEBUG`` is not enabled.